src/intervals/trigonometric.jl

```diff
@@ -27,15 +27,15 @@ end
 function sin{T}(a::Interval{T})
     isempty(a) && return a
 
-    whole_range = Interval(-one(T), one(T))
+    whole_range = Interval{T}(-1, 1)
```

Comment:
I partially agree with your argument 😄. One the one hand, you are right, the code is more readable, the conversion is also there, and calling `one` produces more code. Yet, I can't forget the usual example of type instability where the *extra* conversion saves the day in terms of performance. In any case, maybe for `Interval` the whole discussion is somewhat irrelevant, sorry!, in the sense that we `Interval{Int} is converted by using `float()`. So let's leave it as it is now, and forget my comments on this.